setup.py

```diff
@@ -59,7 +59,7 @@ def run(self):
         if self.type in ['unit', 'all']:
             cmd.append('Ganga/new_tests/*.py')
         if self.type in ['integration', 'all']:
-            cmd.append('Ganga/new_tests/GPI --testmatch="(?:\\b|_)([Tt]est|Savannah|JIRA)"')
+            cmd.append('Ganga/new_tests/GPI --testmatch="(?:\\b|_)(Test|Savannah|JIRA)"')
```

Comment:
Wouldn't this present with no tests being run, yet the tests are running aren't they? (I assume they are simply due to the delay and everything reporting fine)